### DISCUSSION TITLE
How to completely disable the driver and the BMS control?

### DISCUSSION THREAD

**Discussion by erichabg** [Louisvdw/dbus-serialbattery#405] [General]
I have installed the driver for a test.
The installation of the driver has changed the settings of the MPPT controllers to "BMS controlled". So far so good.
After the test I removed the driver with the disabledriver.sh script.
But the MPPTs remain the in "BMS controlled" state and stopped working with error 67 (No BMS).
The control pannel says "Set this back if no bms is present" (translated from German).
But I don't know how to do that?

**Comment by Louisvdw** [comment]
Hi
Follow these steps 
https://github.com/Louisvdw/dbus-serialbattery/wiki/How-to-install#disable-the-driver

**Reply by erichabg** [reply]
Thank You!
In the meanwhile I have found how to do.  I didn't realize that I had to scroll down to get the button "Reset"
Sorry!
